extractor/docker/docker.go

```diff
@@ -98,6 +98,9 @@ func ApplyLayers(layers []types.LayerInfo) types.ImageDetail {
 		}
 
 		for _, pkgInfo := range layer.PackageInfos {
+			for i := range pkgInfo.Packages {
+				pkgInfo.Packages[i].LayerID = layer.ID
+			}
 			nestedMap.SetByString(pkgInfo.FilePath, sep, pkgInfo)
 		}
 		for _, app := range layer.Applications {
```

Comment:
Would you do the same thing for `Applications[].Libraries` as `PackagesInfos[].Packages` here?

```
			for i, libs := range app.Libraries {
				app[i].LayerID = layer.ID
			}
```